rt/src/runtime.rs

```diff
@@ -37,13 +41,11 @@ fn ignore_sigpipe() {
     // While Rust handles this for us when compiling an executable, it doesn't
     // do so when compiling it to a static library and linking it to our
     // generated code, so we must handle this ourselves.
-    unsafe {
-        libc::signal(libc::SIGPIPE, libc::SIG_IGN);
-    }
+    fn signal(sig: i32, handler: SigHandlerT) -> SigHandlerT;
 }
 
 #[cfg(not(unix))]
-fn ignore_sigpipe() {
+fn signal(_: i32, _: SigHandlerT) -> SigHandlerT {
```

Comment:
Sorry for bringing this up late, but this particular conditional function can be removed, as we no longer support Windows, which was the only non-Unix platform we supported.